internal/app/app.go

```diff
@@ -111,6 +113,15 @@ func Run(configPath string) {
 
 	<-quit
 
+	const timeout = 5 * time.Second
```

Comment:
Isn't it better to move the value to the config file?

Reply:
mb, your choise